src/test/kotlin/org/rust/ide/annotator/fixes/AddAsTyFixTest.kt

```diff
@@ -8,7 +8,7 @@ package org.rust.ide.annotator.fixes
 import org.rust.ide.inspections.RsInspectionsTestBase
 import org.rust.ide.inspections.RsTypeCheckInspection
 
-class AddAsTyFixTest : RsInspectionsTestBase(RsTypeCheckInspection()) {
+class AddAsTyFixTest : RsInspectionsTestBase(RsTypeCheckInspection::class.java) {
```

Comment:
Again, maybe it's better to use `RsTypeCheckInspection::class` (and so `KClass`) instead?